engine/evd.mli

```diff
@@ -101,6 +109,10 @@ type evar_info = {
   (** Boolean mask over {!evar_hyps}. Should have the same length.
       When filtered out, the corresponding variable is not allowed to occur
       in the solution *)
+  evar_abstraction : Abstraction.t;
```

Comment:
I think the comment could be improved and the field also renamed to something like `arguments_to_not_imitate`, or `arguments_to_bind`. Plus a pointer to a user (e.g. the pattern tactic or something similar).

Reply:
Ok, I chose evar_abstract_arguments and pointed to second_order_matching